src/explore-education-statistics-common/src/hooks/useMedia.ts

```diff
@@ -4,6 +4,7 @@ export const breakpoints = {
   mobile: '320px',
   tablet: '641px',
   desktop: '769px',
+  desktopWide: '1200px',
```

Comment:
Is this the correct breakpoint? `$dfe-page-width-wide` is 1280px

Reply:
I've removed this now.